Casks/o/opal.rb

```diff
@@ -0,0 +1,34 @@
+cask "opal" do
```

Comment:
```suggestion
cask "opal-app" do
```

We do not allow tokens that overlap with formula in `homebrew-core` - so `opal-app` would be most suitable here. Please also update the filename to match.